package.json

```diff
@@ -102,7 +102,7 @@
     "html-loader": "^2.1.0",
     "html-webpack-plugin": "^5.2.0",
     "jest": "^27.0.6",
-    "node-sass": "^5.0.0",
+    "sass": "1.69.5",
```

Comment:
This was necessary to fix some `npm install` issues due to building native libraries (`node-sass` is deprecated)